lib/cookie.h

```diff
@@ -24,6 +24,7 @@
 #include "curl_setup.h"
 
 #include <curl/curl.h>
+#include "curl_get_line.h"
```

Comment:
This isn't used in the header, is it? I'd rather move it to the implementation file.